docs/WASI-tutorial.md

```diff
@@ -77,9 +77,11 @@ which is configured to target WASI and use the WASI sysroot by default, so we ca
 compile our program like so:
 
 ```
-$ clang demo.c -o demo.wasm
+$ clang demo.c --sysroot <path to sysroot> -o demo.wasm
 ```
 
+The sysroot is folder is located in `opt/wasi-sdk/share/sysroot/`.
```

Comment:
Here, depending on whether 1) you're using the [wasi-sdk](https://github.com/CraneStation/wasi-sdk), or 2) stock `clang-8` with [wasi-sysroot](https://github.com/CraneStation/wasi-sysroot), the path doesn't have to be `opt/wasi-sdk/share/sysroot`. It is true however that by default on Linux, `wasi-sdk` installs into `/opt/wasi-sdk/share/sysroot`, but not necessarily if using the second option. Like above perhaps it'd be better to explain that to the potential reader?

Reply:
Actually, this path [recently changed](https://github.com/CraneStation/wasi-sdk/pull/29), to `/opt/wasi-sdk/share/wasi-sysroot`; would you mind updating the PR here?

Reply:
@sunfishcode this is not released yet right? So I'll stick to `opt/wasi-sdk/share/sysroot/` for now